core/src/main/java/com/adobe/aem/commons/assetshare/content/renditions/download/async/impl/AsyncAssetRenditionsDownloadServlet.java

```diff
@@ -71,16 +71,16 @@
     private static final String DOWNLOAD_ARCHIVE_NAME = PARAM_ARCHIVE_NAME;
 
     @Reference(target="(distribution=cloud-ready)")
-    private RequireAem requireAem;
+    private transient RequireAem requireAem;
 
     @Reference
-    private ActionHelper actionHelper;
+    private transient ActionHelper actionHelper;
```

Comment:
For my edification... the transient keyword I thought was relevant in the context of serialization?

Reply:
yeah - this mostly (AFIAK) to stop Cloud Manager from complaining about https://rules.sonarsource.com/java/RSPEC-1948

You're spot on - technically Servlets are serializable therefore all fields in them should be transient or serializable - since these references wouldn't survive serialization/deserialization even if it were to happen (it won't) - it's better to use `transient` over `serializable` (marking them to NOT be serialized in that event)

It doesn't hurt, is technically correct, and most importantly keeps Cloud Manager code quality quiet :)